atom/browser/api/lib/app.js

```diff
@@ -1,15 +1,21 @@
-var EventEmitter, Menu, app, appPath, bindings, deprecate, downloadItemBindings, fn, i, len, name, ref, ref1, session, wrapDownloadItem,
-  slice = [].slice;
-
-ref = require('electron'), deprecate = ref.deprecate, session = ref.session, Menu = ref.Menu;
-
-EventEmitter = require('events').EventEmitter;
-
-bindings = process.atomBinding('app');
-
-downloadItemBindings = process.atomBinding('download_item');
-
-app = bindings.app;
+const deprecate = require('electron').deprecate;
+const session = require('electron').session;
+const Menu = require('electron').Menu;
+const EventEmitter = require('events').EventEmitter;
+
+const bindings = process.atomBinding('app');
+const downloadItemBindings = process.atomBinding('download_item');
+const app = bindings.app;
+
+var slice = [].slice;
+
+var appPath;
```

Comment:
I think this was the first file we worked on, do we want to inline these now like we ended up doing in the other files?